packages/permission-controller/src/errors.test.ts

```diff
@@ -1,6 +1,26 @@
-import { EndowmentPermissionDoesNotExistError } from './errors';
+import {
+  CaveatMergeTypeMismatchError,
+  EndowmentPermissionDoesNotExistError,
+} from './errors';
 
 describe('error', () => {
+  describe('CaveatMergeTypeMismatchError', () => {
+    it('has the expected shape', () => {
+      expect(new CaveatMergeTypeMismatchError('foo', 'bar').data).toStrictEqual(
+        {
+          leftCaveatType: 'foo',
+          rightCaveatType: 'bar',
+        },
+      );
+    });
+
+    it('does not add an origin property if no data is provided', () => {
+      expect(
+        new EndowmentPermissionDoesNotExistError('bar').data,
```

Comment:
Why are we testing `EndowmentPermissionDoesNotExistError` here?

Reply:
Copypasta error, removed.